source/fundamentals/auth.txt

```diff
@@ -50,14 +50,14 @@ relevant mechanism section and select the :guilabel:`Connection String` tab.
 The ``MongoCredential`` class contains static factory methods that allow you
 to specify your authentication mechanism and credentials. You can use this
 helper class to separate your logic for retrieving and specifying your
-credentials. This helps you avoid having to parse them into a connection
-string, especially when retrieving from external credential providers at
-runtime. If you choose to use the ``MongoCredential`` helper class, you need
-to use the ``MongoClientSettings.Builder`` class to configure your connection
-settings when constructing your ``MongoClient``. For descriptions of how to
-use the ``MongoCredential`` class to specify your authentication method,
-navigate to the relevant mechanism section and select the
-:guilabel:`MongoCredential` tab.
+credentials. This allows you to avoid constructing a connection string with
+your credentials and other connection properties at runtime which is helpful
+when retrieving them from external credential providers. If you choose to use
```

Comment:
I think it's technically accurate now, but needs some more word-smithing.  Perhaps:

    You can use this helper class to separate your logic for retrieving and specifying your
    credentials. For example, you may with to store credentials separately from the rest of 
    the information contained in a connection string, perhaps even retrieving them from an external service
    at runtime.  

Or something like that?

Reply:
I presented the concrete example and tried to directly advise the user and use language that was concise. Hope it is aligned with what you're looking for.